src/app/video-courses/courses/course/date-highlight/date-highlight.directive.ts

```diff
@@ -0,0 +1,25 @@
+import { Directive, HostBinding, Input, OnInit } from '@angular/core';
+
+import { Course } from '../../../../shared/models';
+
+@Directive({
+  selector: '[appDateHighlight]',
+})
+export class DateHighlightDirective implements OnInit {
+  @Input() private course: Course;
+  @HostBinding('style.border') public border: string;
+
+  public ngOnInit(): void {
+    const currentDate: Date = new Date();
+    const { creationDate } = this.course;
+    const twoWeeksMs: number = 14 * 24 * 3600 * 1000;
```

Comment:
is this calculation necessary? Can it be just one value stored into separate constants folder-file?